connect-widget/app/src/pages/connect/components/provider.tsx

```diff
@@ -11,8 +11,8 @@ export type ProviderProps = {
 };
 
 const Provider = (props: ProviderProps) => {
-  const capitalizeFirstLetter = (string: String) => {
-    return string.charAt(0).toUpperCase() + string.slice(1);
+  const capitalizeFirstLetter = (v: string) => {
+    return v.charAt(0).toUpperCase() + v.slice(1);
```

Comment:
nit: `word` would be a better name than `v`

Reply:
It might not be a word. Could use `value`, just looks simpler to use `v` for `value` on one-liners.

I can change if you feel strong about it.

Reply:
Fine for this one liner, but I would err on the side of descriptive variables, unless it's an index.

When I read this code I was thinking "what does it mean by `v`", and `value` or even `str` would be explicit that it's meant to be any string.